DataFormats/HGCalReco/src/classes_def.xml

```diff
@@ -0,0 +1,19 @@
+
+<lcgdict>
+  <class name="ticl::Trackster" ClassVersion="5">
+   <version ClassVersion="5" checksum="3878166595"/>
+  </class>
+  <class name="std::vector<ticl::Trackster>" />
+  <class name="edm::Wrapper<ticl::Trackster>" />
+  <class name="edm::Wrapper<std::vector<ticl::Trackster> >" />
+
+  <class name="TICLLayerTile" ClassVersion="4" persistent="false">
+    <field name="tile_" transient="true"/>
+    <version ClassVersion="4" checksum="63861365"/>
+  </class>
+  <class name="TICLLayerTiles" ClassVersion="4" persistent="false">
```

Comment:
same here: either change to version 3, or, better yet, drop the version support.